packages/eslint-plugin/src/rules/no-extra-parens.ts

```diff
@@ -123,6 +123,19 @@ export default util.createRule<Options, MessageIds>({
       },
       // AssignmentExpression
       // AwaitExpression
```

Comment:
Nit: I think these existing comments were meant to indicate what isn't covered? If so, now that `AwaitExpression` is, no need for this one comment anymore.

```suggestion
```